modules/apps/29-fee/keeper/grpc_query.go

```diff
@@ -1,5 +1,61 @@
 package keeper
 
-// TODO
+import (
+	"context"
 
-//var _ types.QueryServer = Keeper{}
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
```

Comment:
Super small nit: import ordering should be grouped as std lib, external libs, ibc-go imports, [see here](https://github.com/cosmos/ibc-go/pull/337#discussion_r694058339)